cip/1.accepted/CIP2015-06-24-call-procedures.adoc

```diff
@@ -286,6 +297,18 @@ Procedures are a powerful extension mechanism.
 Their introduction opens up new ways of using Cypher which over time may lead to suboptimal usage patterns and hard to read queries.
 The introduction of procedures therefore carries a risk of influencing the long term evolution of the language in a negative way.
 
+
+== Appendix: Procedure naming conventions
+
+It is recommended that procedure namespaces are chosen in accordance with the following conventions:
+
+* `dbms.` Top-level namespace for procedures that operate on the underlying database management system.
+* `db.` Top-level namespace for procedures that operate on the currently selected database.
```

Comment:
The contents of these two name spaces (`db` and `dbms`) should be subject to standard.

Vendor specific procedures should follow the same conventions as user-defined procedures.